CHANGES.md

```diff
@@ -55,6 +55,9 @@
 
 ### Fixes
 
+* The function `xcube.core.resample.resample_in_space()` now supports the parameter
+  `source_ds_subset=True` when calling `rectify_dataset`. This feature enables
+  performing the reprojection exclusively on the congruent subset of the dataset.
```

Comment:
```suggestion
  performing the reprojection exclusively on the spatially congruent subset of 
  the dataset.
```

Reply:
Not sure, maybe _congruent_ applies only to the spatial domain. Then ignore my suggestion.